packages/shared/lib/core/utils/format.utils.ts

```diff
@@ -0,0 +1,22 @@
+import { HEXADECIMAL_PREFIX, HEXADECIMAL_REGEXP } from './constants'
+
+export function formatHexString(
+    hex: string,
+    shouldBeUppercase: boolean = true,
+    shouldIncludePrefix: boolean = true
```

Comment:
Same as above, no need to type booleans if you give the default boolean values. TS is smart enough to know what's happening.